CMakeLists.txt

```diff
@@ -127,6 +127,47 @@ if(MSVC)
   ADD_DEFINITIONS("-DBOOST_ALL_NO_LIB")
 endif(MSVC)
 
+if(RDK_BUILD_INCHI_SUPPORT)
```

Comment:
I'm not thrilled with having this in CMakeLists.txt
If it needs to be separate (and it looks like it does), I would prefer to have this done in $RDBASE/Code/cmake/modules/FindInchi.cmake